README.md

```diff
@@ -119,6 +119,29 @@ Add the language service plugin to the `tsconfig.json` file:
 }
 ```
 
+## Alerts / SLOs
+
+Autometrics makes it easy to add Prometheus alerts using Service-Level Objectives (SLOs) to a function or group of functions.
+
+This works using pre-defined [Prometheus alerting rules](./autometrics.rules.yml) (read more about alerting rules in general [here](https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/)).
```

Comment:
There's a https://github.com/autometrics-dev/autometrics-shared repository that contains the `prometheus.rules.yaml`